packages/components/inputs/select-input/src/select-input.story.js

```diff
@@ -163,6 +163,8 @@ storiesOf('Components|Inputs/SelectInputs', module)
                   value={value}
                   showOptionGroupDivider={showOptionGroupDivider}
                   iconLeft={iconLeft ? createElement(iconLeft) : undefined}
+                  minWidth={number('minWidth', 120)}
```

Comment:
I believe you missed to change the props name here to `minMenuWidth` and `maxMenuWidth`.

But other than that we wanted to have the `horizontalContraints` options for user to choose from to set the min and max width, instead of the pixels, if you remember yesterday's conversation?
But of course let me know if you want to sync some more about it.

Reply:
An example can be `FieldLabel` that uses `horizontalContraints` values for setting the available width of the component.

![Screenshot 2024-01-16 at 14 10 57](https://github.com/commercetools/ui-kit/assets/52276952/8d3c750e-86b8-4cc9-a79c-6ae537cc5530)

Reply:
Oh! very true. 
Thanks for catching this.